posthog/queries/trends/trends.py

```diff
@@ -147,13 +145,16 @@ def _run_query(self, filter: Filter, team: Team, entity: Entity) -> List[Dict[st
         return merged_results
 
     def _run_query_for_threading(
```

Comment:
Problem: This code won't read the right client_query_id since it'll be untagged.

Suggestion: Pass tags to this function, set tags in the function :)

Reply:
![homer-hedge](https://user-images.githubusercontent.com/984817/204664336-195c08a6-6371-4fd0-84cb-16e8025acc7d.gif)